.circleci/config.yml

```diff
@@ -1,39 +1,36 @@
 version: 2
 
 aliases:
-  - &restore_cargo_cache
-    keys:
-      - v2-cargo-{{ checksum "Cargo.lock" }}
-      - cargo-
-
-  - &restore_fixture_cache
-    keys:
-      - fixture-{{ checksum "fixture_setup/Cargo.lock" }}
-      - fixture-
-
-  - &restore_yarn_cache
-    keys:
-      - yarn-{{ checksum "fixture_setup/yarn.lock" }}
-      - yarn-
-
   - &save_cargo_cache
-    key: v2-cargo-{{ checksum "Cargo.lock" }}
+    key: v3-cargo-{{ checksum "Cargo.lock" }}-{{ checksum "fixture_setup/Cargo.lock" }}
     paths:
       - target
+      - fixture_setup/target
       - ~/.cargo
 
-  - &save_fixture_cache
-    key: fixture-{{ checksum "fixture_setup/Cargo.lock" }}
-    paths:
-      - fixture_setup/target
+  - &restore_cargo_cache
+    keys:
+      - v3-cargo-{{ checksum "Cargo.lock" }}-{{ checksum "fixture_setup/Cargo.lock" }}
+      - v3-cargo-
```

Comment:
This should have a middle stage:
```
v3-cargo-{{ checksum "Cargo.lock" }}-
```

Reply:
Why?

Reply:
In case the fixture dependencies change but not the core dependencies.